src/editor/controller/editor_controller.hpp

```diff
@@ -89,7 +89,7 @@ class editor_controller : public controller_base,
 	bool keep_menu_open() const override;
 
 	/** controller_base override */
```

Comment:
This now overrides `command_executor`, assuming you only changed the signature of `controller_base::show_menu`.